packages/salesforcedx-utils-vscode/src/helpers/paths.ts

```diff
@@ -112,13 +113,30 @@ function apexTestResultsFolder(): string {
 }
 
 function apexLanguageServerDatabase(): string {
-  if (!hasRootWorkspace()) {
-    throw new Error(nls.localize('cannot_determine_workspace'));
-  }
```

Comment:
I removed this because:
- the function is only returning a path.  It should be on the caller to verify whether there is a file/resource at that path before doing anything with it
- the single caller for this function currently does just that:

<img width="500" alt="2022-10-13_19-03-20" src="https://user-images.githubusercontent.com/46458081/195746569-c5a5c239-1756-4764-897e-2dee92f31930.png">

Let me know what you think!

Reply:
winner winner